src/views/templates/utils/virtualMachineTemplatesRowFilter.ts

```diff
@@ -0,0 +1,37 @@
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { getTemplateProviderName } from '@kubevirt-utils/resources/template/utils';
+import { FilterValue, RowFilter } from '@openshift-console/dynamic-plugin-sdk';
+
+export const templateProviders = [
+  {
+    id: 'KubeVirt',
+    title: 'KubeVirt',
+  },
+  {
+    id: 'User', // TODO translate?
```

Comment:
@metalice @glekner Do we need translation for this?

Reply:
no